frontend/src/components/Election/VoterAuth.tsx

```diff
@@ -21,6 +21,7 @@ type Props = {
 
 const VoterAuth = ({ authSession, electionData, fetchElection }: Props) => {
   const { voter_id } = useParams();
+  // TODO: maybe we should reconsider useCookie here? this has the potential of inserting the voter id on a different election
```

Comment:
I think we should consider removing the voter cookie in a future PR, curious to hear feedback on this

Reply:
I'd be interested in alternatives. Cookies are a convenient place to store the voter_id.

Reply:
my bad, I forgot that we needed to keep the voter id for the scope of the whole voting session. Cookie is fine for that case

And as long as we're clearing the cookie when id isn't in the url (which it seems like it does), then that's fine